modules/runner/src/test/java/org/apache/ignite/internal/configuration/storage/LocalFileConfigurationStorageTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.configuration.storage;
+
+import java.nio.file.Path;
+import java.util.Random;
+import org.apache.ignite.internal.configuration.NodeBootstrapConfiguration;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Tests for the {@link LocalFileConfigurationStorage}.
+ */
+public class LocalFileConfigurationStorageTest extends ConfigurationStorageTest {
+
+    @TempDir
```

Comment:
I wonder why you don't use `@WorkDir` (or whatever that name is, it's our own path provider)